tests/Command/CommandTest.php

```diff
@@ -85,8 +85,10 @@ public function testRunCommandWithoutOptionsAndNotReuseKernel(): void
         $this->assertIsBool($this->getDecorated());
         $this->assertTrue($this->getDecorated());
 
-        // Run command and not reuse kernel
-        $this->environment = 'prod';
```

Comment:
Please duplicate this test to use `$this->environment = 'prod';` and not `self::$env = 'prod';`, to ensure BC.

Reply:
done